apps/challenges/serializers.py

```diff
@@ -63,11 +73,47 @@ class ChallengePhaseSplitSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = ChallengePhaseSplit
-        fields = '__all__'
         fields = ('id', 'dataset_split', 'challenge_phase', 'challenge_phase_name', 'dataset_split_name', 'visibility')
 
     def get_dataset_split_name(self, obj):
         return obj.dataset_split.name
 
     def get_challenge_phase_name(self, obj):
         return obj.challenge_phase.name
+
+
+class DatasetSplitSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = DatasetSplit
+        fields = '__all__'
+
+
+class LeaderboardSerializer(serializers.ModelSerializer):
+    """
+    Serialize the Leaderboard Model.
+    """
+    class Meta:
+        model = Leaderboard
+        fields = '__all__'
+
+
+class ZipConfigurationChallengeSerializer(ChallengeSerializer):
+    """
+    Serialize the Challenge Model and is used to create challenge using zip file uploaded by user.
```

Comment:
Can we rewrite the doc string as "Serializer used for creating challenge through zip file". Same goes for ZipFileCreateChallengePhaseSplitSerializer.